docs/source/developing-volttron/developing-agents/using-asyncio-in-agents.rst

```diff
@@ -53,9 +40,24 @@ called '_start_asyncio_loop', which will be used to create tasks from the aforem
         loop.run_forever()
 
 
-To use the Event Loop within the Volttron Agent Framework, we will use the Volttron feature, :ref:`Platform Configuration Store<Agent-Configuration-Store>`.
-In the constructor of our agent, our agent will subscribe to changes in an agent via a callback.
-Below is an example of the constructor:
+Next, the method, 'self._configure_ven_client', contains code to configure our agent including running the Event Loop defined above.
+When Volttron starts the agent, it will call this method. This method is where we use Gevent with Asyncio. Gevent will spawn a coroutine
+(i.e. `greenlet <https://greenlet.readthedocs.io/en/latest/greenlet.html#what-is-greenlet>`_); the Event Loop from Asyncio will be run
+inside that coroutine. Below is the relevant snippet of the callback function:
+
+.. code-block:: python
+
+    def _configure_ven_client(self, config_name, action, contents):
+        # do some configuration
+        ...
+
+        gevent.spawn_later(3, self._start_asyncio_loop)
+
+
+The constructor of this agent will include this method, 'self._configure_ven_client'.
+To use the Event Loop within the Volttron Agent Framework, the agent's Configuration Store will subscribe to changes in
+an agent's configuration and use that method as the callback function (i.e. the function to be called in response to changes
```

Comment:
I would stop using Event Loop without a qualifying gevent or async.  B/C with volttron you really don't care about the event loop at all.  Its there but doesn't get in the way.

We Async in a box (greenlet) and say you play over here until you have something we want you to do.  That's what the spawn_later does...runs that function in a greenlet.

Reply:
Ok. I'll update the verbiage to remove Event Loop in this section and update the verbiage accordingly.